src/wallet3/cli-wallet/.gitignore

```diff
@@ -4,3 +4,4 @@ venv/
 build/
 release/
 dist/
+testnet-oxen-wallet.s*
```

Comment:
should probably go in a local/private `.gitignore` as it seems to be an ad-hoc local file for testing.